packages/global/scss/components/_content-page.scss

```diff
@@ -1,8 +1,17 @@
 .page-contents {
   .primary-image {
-    &__image-credit:before {
-      content: "";
-      display: none;
```

Comment:
Was this intended to be removed? I thought the reason this was added is so they didn't have "Source: Photo:..." showing up on these.